client/landing/stepper/declarative-flow/internals/components/google-domains-transfer-instructions/index.tsx

```diff
@@ -16,6 +17,11 @@ const GoogleDomainsModal: React.FC< Props > = ( { children, className, focusedSt
 	const [ isOpen, setOpen ] = useState( false );
 	const openModal = () => setOpen( true );
 	const closeModal = () => setOpen( false );
+	const step2Text = useHasEnTranslation()(
+		'Select the domain you want to transfer in the "My domains" section.'
+	)
+		? __( 'Select the domain you want to transfer in the "My domains" section.' )
+		: __( "Click on the name of the domain that you'd like to transfer." );
```

Comment:
Should we add, 'in the "My domains" section.'

Reply:
Yep, it sounds better @ebinnion, updated!